Myfeature

baqar-abbas opened the pull request:
- Create a Pull request 
- Saved all the files here. Incorporated the suggested changes recommended by the code reviewer.
-  updated CSS file for managing spaces and indentations.

baqar-abbas committed:
Create styles.css

Created CSS file and give a text color

baqar-abbas committed:
Updated .hintrc .stylelintrc npm json package updated and saved

Updated .hintrc .stylelintrc npm json package updated and saved

abdulhamiid commented on styles.css:
```diff
@@ -0,0 +1,3 @@
+h1 {
+ color: blue; 
+}
```
![image](https://user-images.githubusercontent.com/78251927/211651701-5dc44512-b71b-4cb0-8432-c5d584865fa8.png)

Please kindly fix the Linters / Stylelint error, all linter results should be green ✔️. AKA: Linter should report NO errors.

To view the report in your terminal, Run  `npx stylelint "**/*.{css,scss}"`.  then you can solve the error 

Or automatically fix by using `npx stylelint "**/*.{css,scss} --fix"`

baqar-abbas committed:
Update styles.css

Indentation added

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @baqar-abbas 👋 , 
![Alt Text](https://c.tenor.com/vOLW_UQMySIAAAAC/almost-there.gif)
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️

- [ ] Please kindly customize all entries of your project **README file**, The readme file is used to explain our project and how we can install or use it for other developers, so let's make it more descriptive as possible. 💪 

- [ ] Please kindly update your Pull request summary
![image](https://user-images.githubusercontent.com/78251927/211656236-18d7c255-0b34-4d9a-82a2-69873cc94283.png)

  - Pull Request should have a descriptive title telling you what changes are inside.
      
   - Pull Request should have a summary describing introduced changes in more detail. please let's fix that. 🤗

- [ ] Please make sure to include a LICENSE to your project, at the moment this is not working. Microverse suggests you use the MIT one. Follow the [official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to achieve this! 🤓

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

baqar-abbas committed:
Update styles.css

Added 2 spaces Indentation and new line added

baqar-abbas committed:
Remove space added 2 space indentation 

Remove space added 2 space indentation

baqar-abbas committed:
updated srtles.css

indendation and spaces updated in the styles.css file and saved the changes made to them.

baqar-abbas committed:
Merge branch 'myfeature' of https://github.com/baqar-abbas/Hello_World_Microverse into myfeature

baqar-abbas committed:
Updated Readme file with more details as suggested by the Code reviewer

baqar-abbas committed:
Updated Readme file with more decriptions as suggested by the Code reviewer

baqar-abbas committed:
Spell Check and corrected

TuyishimireEric commented on README.md:
```diff
@@ -268,13 +234,13 @@ I would like to thank...
+- **[Did you understand the Git Hub Process Flow]**
 
-  - [Answer_1]
+  - [Yes and will improve as I gain more experience working with Git Hub]
 
-- **[Question_2]**
+- **[Do you understand Git Process Flow]**
 
-  - [Answer_2]
+  - [Yes and will improve as I gain more experience working with Git]
```
- [ ]  suggested by the previous Reviewer, Kindly provide a valid link for your project's License we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

TuyishimireEric reviewed (changes requested):
# Status: REQUIRED CHANGES :recycle: 
### Hi  @baqar-abbas     👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

## Highlights 
✔️No Linter Error 👍

## Required Changes ♻️

- [ ] **Kindly as suggested by the previous reviewer,** It would be better to make your Pull Request title more descriptive. You may rename it to the performed action and this will make it more Professional 👍😎


**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ABDULALI3468 reviewed (approved):
# APPROVED 🥇 

Hi @baqar-abbas , 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media2.giphy.com/media/3oz9ZE2Oo9zRC/giphy.gif"/>

## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: ![GitHub followers](https://img.shields.io/github/followers/ABDULALI3468?label=ABDULALI&style=social)
> * This is My **Linkedin**: [![Linkedin Badge](https://img.shields.io/badge/-ABDUL%20ALI-blue?style=flat-square&logo=Linkedin&logoColor=white&link=https://www.linkedin.com/in/abdul-ali-5400bb216/)](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [![Twitter Badge](https://img.shields.io/badge/-@mrabdul_ali_-1ca0f1?style=flat-square&labelColor=1ca0f1&logo=twitter&logoColor=white&link=https://twitter.com/mrabdul_ali)](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

baqar-abbas commented:
Thanks for the valuable feedback I am merging the Pull Request after your Approval.
Regards, Baqar Abbas
